libgalois/include/galois/analytics/connected_components/connected_components.h

```diff
@@ -111,14 +110,6 @@ class ConnectedComponentsPlan : Plan {
         kCPU, kEdgeAfforest, edge_tile_size, neighbor_sample_size,
         component_sample_frequency};
   }
-
-  static ConnectedComponentsPlan Automatic(
-      ptrdiff_t edge_tile_size = 512, uint32_t neighbor_sample_size = 2,
-      uint32_t component_sample_frequency = 1024) {
-    return {
-        kCPU, kAfforest, edge_tile_size, neighbor_sample_size,
-        component_sample_frequency};
-  }
```

Comment:
We can use the default constructor as the automatic constructor if you like, but it's not what I did in other cases. In other cases there was an `Automatic` function that configured the algorithm automatically. If we want to switch, we should do it across the board. I encourage you to look at BFS: https://github.com/KatanaGraph/katana/blob/master/libgalois/include/galois/analytics/bfs/bfs.h#L28-L45